troubleshootingneuralnets.md

```diff
@@ -61,6 +61,12 @@ The usual approach to selecting an appropriate learning rate is to use [DL4J's v
 
 For training neural networks in a distributed manner, you may need a different (frequently higher) learning rate compared to training the same network on a single machine.
 
+### Policies and Scheduling
+
+You can optionally define a learning rate policy for your neural network. A policy will change the learning rate over time, achieving better results since the learning rate can "slow down" to find a closer local minima for convergence. A common policy used is scheduling. See the [LeNet example](https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/convolution/LenetMnistExample.java) for a learning rate schedule used in practice.
```

Comment:
The indefinite article a may not be required with the plural noun minima in this sentence. Consider removing the article, or changing the noun to singular.